### PR TITLE
06-3.md Korean Translation Update

### PR DESCRIPTION
**Pull request by shc443**

### CODE REVIEW

File: docs/ko/week06/06-3.md [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+</center>
+-->
+<center>
+<img src="{{site.baseurl}}/images/week06/06-3/rnn_3.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>Figure 14:</b> 소실 문제점
+</center>
+
+<!--The figure above is a typical RNN architecture. In order to perform rotation over previous steps in RNN, we use matrices, which can be regarded as horizontal arrows in the model above. Since the matrices can change the size of outputs, if the determinant we select is larger than 1, the gradient will inflate over time and cause gradient explosion. Relatively speaking, if the eigenvalue we select is small across 0, the propagation process will shrink gradients and leads to the gradient vanishing.
+-->
+위의 도표는 전형적인 RNN 구조이다. 전 단계의 RNN의 회전을 실행하기 위해서는 위 모델에서의 오른쪽 방향의 화살표로 표현되는 행렬이 사용된다. 행렬은 출력값의 크기를 바꿀 수 있기 때문에, 행렬식<sup>determinant</sup>이 1보다 클 경우, 경사도는 시간이 계속 될수록 점점 증가하게 되며, 결국엔 경사도의 폭증을 야기한다. 비슷한 맥락으로, 만일 고유치<sup>eigenvalue</sup>가 0에 가깝게 작다면 전파과정에는 경사도를 지속적으로 줄어들게하며, 결국엔 경사도의 소실을 불러온다.
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:339]:
```suggestion
위의 도표는 전형적인 RNN 구조이다. RNN에서 이전 단계에 대한 회전을 실행하기 위해서는, 위 모델에서의 오른쪽 방향의 화살표로 표현되는 행렬이 사용된다. 행렬은 출력의 크기를 바꿀 수 있기 때문에, 행렬식<sup>determinant</sup>이 1보다 클 경우, 경사도는 시간이 지속될수록 점점 증가하게 되며, 결국엔 경사도의 폭증을 야기한다. 비슷한 맥락으로, 만일 고유치<sup>eigenvalue</sup>가 0에 가까울 정도로 작다면, 전파과정에는 경사도가 지속적으로 줄어들게 되며, 결국엔 경사도의 소실을 불러온다.
```

File: docs/ko/week06/06-3.md [hunk 2]
```diff
@@ -0,0 +1,580 @@
...
+
+<!--In typical RNNs, gradients will be propagated through all the possible arrows, which provides the gradients a large chance to vanish or explode. For example, the gradient at time 1 is large, which is indicated by the bright color. When it goes through one rotation, the gradient shrinks a lot and at time 3, it gets killed.
+-->
+일반적인 RNN에서 경사도는 받아들일 수 있는 모든 화살표를 통해 전파되어 높은 가능성으로 경사도를 소신시키거나 폭증시킨다. 예를 들어 시간 1에서의 경사도가 매우 크다면 (배우 밝은색을 의미한다) 단 한 번의 회전을 통해 경사도는 매우 크게 줄어들게 되며, 시간 3에서는 결국 사라지게될 것이다.
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:343]:
```suggestion
일반적인 RNN에서 경사도는 받아들일 수 있는 모든 화살표를 통해 전파되어, 경사도를 소신시키거나 폭증시킬 가능성이 높다. 예를 들어 시간 1에서의 경사도가 매우 크다면,(그림에서는 밝은색을 의미한다) 단 한 번의 회전을 통해, 경사도는 매우 크게 줄어들게 되며, 시간 3에서는 결국 사라지게 될 것이다. 
```

File: docs/ko/week06/06-3.md [hunk 3]
```diff
@@ -0,0 +1,580 @@
...
+-->
+일반적인 RNN에서 경사도는 받아들일 수 있는 모든 화살표를 통해 전파되어 높은 가능성으로 경사도를 소신시키거나 폭증시킨다. 예를 들어 시간 1에서의 경사도가 매우 크다면 (배우 밝은색을 의미한다) 단 한 번의 회전을 통해 경사도는 매우 크게 줄어들게 되며, 시간 3에서는 결국 사라지게될 것이다. 
+
+<!--### Solution
+-->
+### 해결방안
+
+<!--An ideal to prevent gradients from exploding or vanishing is to skip connections. To fulfill this, multiply networks can be used.
+-->
+경사도의 소실과 폭증을 막기 위한 이상적인 방법은 연결을 건너 뛰는 것이다. 이와 같은 과정을 실행하기 위해서 신경곱<sup>multiply networks</sup>이 사용되어진다
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:351]:
```suggestion
경사도의 소실과 폭증을 막기 위한 이상적인 방법은 연결을 건너뛰는 것<sup>skip connection</sup>이다. 이와 같은 과정을 실행하기 위해서, 신경곱<sup>multiply networks</sup>이 사용된다.
```

File: docs/ko/week06/06-3.md [hunk 4]
```diff
@@ -0,0 +1,580 @@
...
+</center>
+-->
+<center>
+<img src="{{site.baseurl}}/images/week06/06-3/rnn_2.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>Figure 15:</b> 연결 건너뛰기
+</center>
+
+<!--In the case above, we split the original network into 4 networks. Take the first network for instance. It takes in a value from input at time 1 and sends the output to the first intermediate state in the hidden layer. The state has 3 other networks where the $\circ$s allows the gradients to pass while the $-$s blocks propagation. Such a technique is called gated recurrent network.
+-->
+위와 같은 경우 원래의 신경망을 4개로 나눈다. 첫번째 신경망을 예로 시간 1에서의 입력값을 갖고 출력값을 은닉망의 처음 중간상태로 보낸다. 이 상태는 경사도를 통과시키는$\circ$와 경사도를 막는 $-$로 이루어진 3개의 다른 신경망을 갖는다. 이러한 기법을 게이트 순환 신경망 <sup>gated recurrent network</sup>이라고 한다.
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:365]:
```suggestion
위와 같은 경우, 기존의 신경망을 4개 영역으로 나눈다. 첫번째 신경망을 보면, 시간 1에서의 입력값을 갖고 출력값을 은닉망의 처음 중간상태로 보낸다. 이 상태는 경사도를 통과시키는$\circ$와 경사 전파를 막는 $-$로 이루어진 3개의 다른 신경망으로 구성된다. 이러한 기법을 게이트 순환 신경망 <sup>gated recurrent network</sup>이라고 한다.
```

File: docs/ko/week06/06-3.md [hunk 5]
```diff
@@ -0,0 +1,580 @@
...
+
+
+<!--### Model Architecture
+-->
+### 모델 구조
+
+
+<!--Below are equations expressing an LSTM. The input gate is highlighted by yellow boxes, which will be an affine transformation. This input transformation will be multiplying $c[t]$, which is our candidate gate.
+-->
+아래는 LSTM을 표현하는 공식이다. 노란색 박스 안에 있는 공식들이 아핀변환<sup>affine transformation</sup>이 될 입력 게이트로 입력변환은 후보 게이트인 $c[t]$를 곱함으로 진행된다.
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:384]:
```suggestion
아래는 LSTM을 표현하는 공식이다. 노란색 박스 안에 있는 공식들이 아핀변환<sup>affine transformation</sup>이 될 입력 게이트로, 이 입력변환은 후보 게이트인 $c[t]$를 곱함으로 진행된다.
```

File: docs/ko/week06/06-3.md [hunk 6]
```diff
@@ -0,0 +1,580 @@
...
+-->
+### 시퀀스 분류<sup>Sequence Classification</sup>
+
+<!--The goal is to classify sequences. Elements and targets are represented locally (input vectors with only one non-zero bit). The sequence **b**egins with an `B`, **e**nds with a `E` (the “trigger symbol”), and otherwise consists of randomly chosen symbols from the set `{a, b, c, d}` except for two elements at positions $t_1$ and $t_2$ that are either `X` or `Y`. For the `DifficultyLevel.HARD` case, the sequence length is randomly chosen between 100 and 110, $t_1$ is randomly chosen between 10 and 20, and $t_2$ is randomly chosen between 50 and 60. There are 4 sequence classes `Q`, `R`, `S`, and `U`, which depend on the temporal order of `X` and `Y`. The rules are: `X, X -> Q`; `X, Y -> R`; `Y, X -> S`; `Y, Y -> U`.
+-->
+
+시퀀스를 분류하기 위해 요소들과 목표들은 로컬에 표현됩니다 (단 하나만의 논제로 비트<sup>non-zero bit</sup>인 입력벡터). 시퀀스는  `B`로 부터 시작<sup>**B**egin</sup>  하여, `E`로 끝<sup>**E**nd</sup>이나고, 그 외에는  $t_1$와 $t_2$에 위치한 `X` 나 `Y`를 제외하고 `{a, b, c, d}` 세트 내에서 임의로 선택되어진다. `DifficultyLevel.HARD` 의 경우, 시퀀스 길이는 100과 110 사이로 임의의 수로 정해지고, $t_1$은 10에서 20, $t_2$는 50에서 60 사이의 숫자 중 임의로 결정된다. 그 다음으로는 총 4개의 시퀀스`Q`, `R`, `S`, `U`가 존재하는데
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:483]:
```suggestion
시퀀스를 분류하기 위해 요소들과 목표들은 로컬에 표현된다 (단 하나만의 0이 아닌 비트<sup>non-zero bit</sup>인 입력벡터). 시퀀스는  `B`로 부터 시작하여, `E`로 끝나고("트리거 심볼"이라고 표현한다.), 그 외에는  $t_1$와 $t_2$에 위치한 `X` 나 `Y`를 제외하고 `{a, b, c, d}` 세트 내에서 임의로 선택된다. `DifficultyLevel.HARD` 의 경우, 시퀀스 길이는 100과 110 사이로 임의의 수로 정해지고, $t_1$은 10에서 20, $t_2$는 50에서 60 사이의 숫자 중 임의로 결정된다. 그 다음으로는 총 4개의 시퀀스인 `Q`, `R`, `S`, `U`가 존재하는데
```

File: docs/ko/week06/06-3.md [hunk 7]
```diff
@@ -0,0 +1,580 @@
...
+
+
+<!--1). Dataset Exploration
+-->
+1). 데이터셋 탐사
+
+<!--The return type from a data generator is a tuple with length 2. The first item in the tuple is the batch of sequences with shape $(32, 9, 8)$. This is the data going to be fed into the network. There are eight different symbols in each row (`X`, `Y`, `a`, `b`, `c`, `d`, `B`, `E`). Each row is a one-hot vector. A sequence of rows represents a sequence of symbols. The first all-zero row is padding. We use padding when the length of the sequence is shorter than the maximum length in the batch.  The second item in the tuple is the corresponding batch of class labels with shape $(32, 4)$, since we have 4 classes (`Q`, `R`, `S`, and `U`). The first sequence is: `BbXcXcbE`. Then its decoded class label is $[1, 0, 0, 0]$, corresponding to `Q`.
+-->
+데이터 생성기에서의 리턴 타입은 길이 2의 투플<sup>tuple</sup>이며 투플의 첫번째 아이템은 신경망에 피드될 $(32, 9, 8)$의 크기를 가진 시퀀스 이다. 각 열에는 8가지의 원-핫 벡터인 심볼들이(`X`, `Y`, `a`, `b`, `c`, `d`, `B`, `E`) 존재하며, 시퀀스 열들은 심볼 시퀀스를 표현한다. 모두 0인 첫번째 열은 패딩으로 패딩은 시퀀스의 길이가 배치의 최장 길이보다 짧을 때 쓰여진다. 두번째 아이템은 클래스 라벨 배치로  (`Q`, `R`, `S`,  `U`) 4개의 클래스를 갖기에 $(32, 4)$의 크기를 갖는다.  첫번째 시퀀스는 `BbXcXcbE`로 해석된 클래스 라벨은 $[1, 0, 0, 0]$로 `Q`와 일치한다.
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:493]:
```suggestion
데이터 생성기에서의 리턴 타입은 길이 2의 투플<sup>tuple</sup>이며 투플의 첫번째 아이템은 신경망에 피드될 $(32, 9, 8)$의 크기를 가진 시퀀스이다. 각 행에는 8가지의 원-핫 벡터인 심볼들이(`X`, `Y`, `a`, `b`, `c`, `d`, `B`, `E`) 존재하며, 시퀀스 행들은 심볼 시퀀스를 표현한다. 모두 0인 첫번째 열은 패딩으로, 패딩은 시퀀스의 길이가 배치의 최장 길이보다 짧을 때 쓰여진다. 튜플의 두번째 아이템은 클래스 라벨 배치로,  (`Q`, `R`, `S`,  `U`) 4개의 클래스를 갖기 때문에 $(32, 4)$의 크기를 갖는다.  첫번째 시퀀스는 `BbXcXcbE`일 경우, 해석된 클래스 라벨은 $[1, 0, 0, 0]$로 `Q`와 일치한다.
```

File: docs/ko/week06/06-3.md [hunk 8]
```diff
@@ -0,0 +1,580 @@
...
+
+<!--Let’s create a simple recurrent network, an LSTM, and train for 10 epochs. In the training loop, we should always look for five steps:
+
+ * Perform the forward pass of the model
+ * Compute the loss
+ * Zero the gradient cache
+ * Backpropagate to compute the partial derivative of loss with regard to parameters
+ * Step in the opposite direction of the gradient
+-->
+심플한 RNN과 LSTM을 만들어 10번의 에폭<sup>epoch</sup>동안 학습시켜봅시다. 학습 루프 동안에는, 다음과 같이 5개의 단계를 꼭 확인해야합니다:
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:518]:
```suggestion
심플한 RNN과 LSTM을 만들어 10번의 에폭<sup>epoch</sup>동안 학습시켜보자. 학습 루프 동안에는, 다음과 같이 5개의 단계를 꼭 확인해야한다:
```

File: docs/ko/week06/06-3.md [hunk 9]
```diff
@@ -0,0 +1,580 @@
...
+-->
+<center>
+<img src="{{site.baseurl}}/images/week06/06-3/train_test_easy.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>Figure 24:</b>심플한 RNN vs LSTM - 10 에폭
+</center>
+
+
+<!--With an easy level of difficulty, RNN gets 50% accuracy while LSTM gets 100% after 10 epochs. But LSTM has four times more weights than RNN and has two hidden layers, so it is not a fair comparison. After 100 epochs, RNN also gets 100% accuracy, taking longer to train than the LSTM.
+-->
+쉬운 난이도에서는 10 에폭 이후 RNN는 50%의 정확도를 보인 반면, LSTM은 100%의 정확도를 가졌다. 하지만 LSTM은 RNN보다 4배나 많은 수의 가중치를 가졌고 은닉망을 두개를 보유하였기에 같은 선상에서 비교를 하는 것은 무리가 있다. 100 에폭 후에는 LSTM보다 학습하는 데에 더 많은 시간을 소요하였으나 RNN 또한 100%의 정확도를 가졌다.
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:539]:
```suggestion
쉬운 난이도에서는 10 에폭 이후 RNN는 50%의 정확도를 보인 반면, LSTM은 100%의 정확도를 가졌다. 하지만 LSTM은 RNN보다 4배나 많은 수의 가중치를 가졌고, 은닉망도 두 개를 보유하였기 때문에, 같은 조건상에서 비교를 하는 것은 무리가 있다. 100 에폭 후에는, RNN 또한 100%의 정확도를 가졌지만, LSTM을 학습하는 때보다 더 많은 시간이 걸렸다.
```

File: docs/ko/week06/06-3.md [hunk 10]
```diff
@@ -0,0 +1,580 @@
...
+</center>
+-->
+<center>
+<img src="{{site.baseurl}}/images/week06/06-3/train_test_hard.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>Figure 25:</b>심플한 RNN vs LSTM - 100 에폭
+</center>
+
+<!--If we increase the difficulty of the training part (using longer sequences), we will see the RNN fails while LSTM continues to work.
+-->
+더욱 긴 시퀀스를 사용하여 학습부분에서의 난이도를 증가시키면 RNN은 실패하나 LSTM은 계속 돌아가는 현상을 볼 수 있다.
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:553]:
```suggestion
더욱 긴 시퀀스를 사용하여 학습부분에서의 난이도를 증가시키면, RNN은 실패하나 LSTM은 계속 돌아가는 것을 확인할 수 있다. 
```

File: docs/ko/week06/06-3.md [hunk 11]
```diff
@@ -0,0 +1,580 @@
...
+</center>
+-->
+<center>
+<img src="{{site.baseurl}}/images/week06/06-3/hidden_state_lstm.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<b>Figure 26:</b>은닉 상태 값 시각화<sup>Visualization of Hidden State Value</sup> 
+</center>
+
+<!--The above visualization is drawing the value of hidden state over time in LSTM. We will send the inputs through a hyperbolic tangent, such that if the input is below $-2.5$, it will be mapped to $-1$, and if it is above $2.5$, it will be mapped to $1$. So in this case, we can see the specific hidden layer picked on `X` (fifth row in the picture) and then it became red until we got the other `X`. So, the fifth hidden unit of the cell is triggered by observing the `X` and goes quiet after seeing the other `X`. This allows us to recognize the class of sequence.
+-->
+위 도표는 LSTM 은닉 상태 값이 시간에 따라 어떻게 바뀌는지 보여준다. 입력값을 쌍곡 탄잰트에 넣어 입력값이  $-2.5$ 이하면 $-1$ 매핑하고 $2.5$ 이상이면 $1$로 매핑한다. 이 경우, 우리는 특정 은닉망이 (그림에서의 5번째 열에서) `X`를 선택하고 그 다음  `X`가 나오기 전까지 값들을 붉은색으로 바꾼다. 그러므로 5번째 셀의 은닉 유닛이 `X`를 관측 후 촉발되었고, 다른  `X`를 확인 후 잠잠해졌다. 이런 과정은 시퀀스의 클래스를 인식하는 것을 가능케 한다.
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:567]:
```suggestion
위 도표는 LSTM 은닉 상태 값이 시간에 따라 어떻게 바뀌는지 보여준다. 입력값을 쌍곡 탄잰트에 넣어 입력값이  $-2.5$ 이하면 $-1$ 매핑하고 $2.5$ 이상이면 $1$로 매핑한다. 이 경우, 우리는 특정 은닉망이 (그림에서의 5번째 행에서) `X`를 선택하고, 그 다음  `X`가 나오기 전까지 값들을 붉은색으로 바꾼다. 그러므로 5번째 셀의 은닉 유닛이 `X`를 관측 후 촉발되었고, 다른  `X`를 확인 후 잠잠해졌다. 이런 과정은 시퀀스의 클래스를 인식하는 것을 가능케 한다. 
```

File: docs/ko/week06/06-3.md [hunk 12]
```diff
@@ -0,0 +1,580 @@
...
+위 도표는 LSTM 은닉 상태 값이 시간에 따라 어떻게 바뀌는지 보여준다. 입력값을 쌍곡 탄잰트에 넣어 입력값이  $-2.5$ 이하면 $-1$ 매핑하고 $2.5$ 이상이면 $1$로 매핑한다. 이 경우, 우리는 특정 은닉망이 (그림에서의 5번째 열에서) `X`를 선택하고 그 다음  `X`가 나오기 전까지 값들을 붉은색으로 바꾼다. 그러므로 5번째 셀의 은닉 유닛이 `X`를 관측 후 촉발되었고, 다른  `X`를 확인 후 잠잠해졌다. 이런 과정은 시퀀스의 클래스를 인식하는 것을 가능케 한다. 
+
+<!--### Signal Echoing
+-->
+### 에코 신호<sup>Signal Echoing
+</sup>
+
+<!--Echoing signal n steps is an example of synchronized many-to-many task. For instance, the 1st input sequence is `"1 1 0 0 1 0 1 1 0 0 0 0 0 0 0 0 1 1 1 1 ..."`, and the 1st target sequence is `"0 0 0 1 1 0 0 1 0 1 1 0 0 0 0 0 0 0 0 1 ..."`. In this case, the output is three steps after. So we need a short-time working memory to keep the information. Whereas in the language model, it says something that hasn't already been said.
+-->
+신호를 n번 에코를 주는건 동시 다대다 작업<sup>synchronized many-to-many</sup>의 한 예이다. 예를 들어 첫번째 입력 시퀀스가 `"1 1 0 0 1 0 1 1 0 0 0 0 0 0 0 0 1 1 1 1 ..."` 이고 목표 시퀀스가 `"0 0 0 1 1 0 0 1 0 1 1 0 0 0 0 0 0 0 0 1 ..."`라고 해보자. 이런 경우에는 출력은 3번의 스텝 이 후이기에 정보를 저장할 단기 메모리가 필요하다. 하지만 언어 모델같은 경우에는 아직까지 말하지 않은 말들을 하게된다.
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:576]:
```suggestion
신호를 n번 에코를 주는 것은 동기화된 다대다 작업<sup>synchronized many-to-many</sup>의 한 예이다. 예를 들어 첫번째 입력 시퀀스가 `"1 1 0 0 1 0 1 1 0 0 0 0 0 0 0 0 1 1 1 1 ..."` 이고 목표 시퀀스가 `"0 0 0 1 1 0 0 1 0 1 1 0 0 0 0 0 0 0 0 1 ..."`라고 해보자. 이런 경우에는 출력은 3번의 시간 단계에 나오기 때문에, 이런 정보를 저장할 단기 메모리가 필요하다. 하지만 언어 모델같은 경우에는 아직까지 말하지 않은 말들을 하게 된다. 
```

File: docs/ko/week06/06-3.md [hunk 13]
```diff
@@ -0,0 +1,580 @@
...
+-->
+신호를 n번 에코를 주는건 동시 다대다 작업<sup>synchronized many-to-many</sup>의 한 예이다. 예를 들어 첫번째 입력 시퀀스가 `"1 1 0 0 1 0 1 1 0 0 0 0 0 0 0 0 1 1 1 1 ..."` 이고 목표 시퀀스가 `"0 0 0 1 1 0 0 1 0 1 1 0 0 0 0 0 0 0 0 1 ..."`라고 해보자. 이런 경우에는 출력은 3번의 스텝 이 후이기에 정보를 저장할 단기 메모리가 필요하다. 하지만 언어 모델같은 경우에는 아직까지 말하지 않은 말들을 하게된다. 
+
+<!--Before we send the whole sequence to the network and force the final target to be something, we need to cut the long sequence into little chunks. While feeding a new chunk, we need to keep track of the hidden state and send it as input to the internal state when adding the next new chunk. In LSTM, you can keep the memory for a long time as long as you have enough capacity. In RNN, after you reach a certain length, it starts to forget about what happened in the past.
+-->
+시퀀스 전부를 신경망에 보내고 최종 목표를 정하기 전에, 긴 시퀀스를 작은 조각들로 나누어야 한다. 새로운 조각을 피드하는 중에도 은닉 상태를 계속 확인 하면서  내부 상태에 입력으로 보내야한다. LSTM에서는 메모리를 메모리 한계가 되는한 오랜시간 보존할 수 있다. 하지만 RNN의 경우 특정 길이 이상으로는, 과거의 있던 메모리를 잊기 시작한다.
```
Comment (goodboychan) [inline docs/ko/week06/06-3.md:580]:
```suggestion
시퀀스 전부를 신경망에 보내고 최종 목표를 정하기 전에, 긴 시퀀스를 작은 조각들로 나누어야 한다. 새로운 조각을 피드하는 중에도 은닉 상태를 계속 확인하면서, 다음 새로운 조각이 추가될 때 이를 내부 상태의 입력으로 보내야한다. LSTM에서는 메모리 공간이 충분한 동안에는 메모리의 정보를 오랜시간 보존할 수 있다. 하지만 RNN의 경우, 특정 길이 이상으로는, 과거의 있던 메모리를 잊기 시작한다. 
```

**Comment by Atcold** [comment]
@skyil7, I'm leaving this to you, okay?

**Comment by skyil7** [comment]
Sure, @Atcold.

**Pull request merged**